src/main/java/core/basesyntax/RandomSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomSupplier {
```

Comment:
no need to create this class
Why `randomSupplier.getRandomValue(xxx)` better then `random.nextInt(xxx)`?